### PR TITLE
Database/mhmd

### PR DESCRIPTION
**Pull request by MhmdIbrahim1**
first edit, please accept

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <TextView
```
Comment (MostafaMohamed2002) [inline app/src/main/res/layout/activity_main.xml:9]: increase font size

Comment (MhmdIbrahim1) [reply app/src/main/res/layout/activity_main.xml:9]: okay

Comment (MhmdIbrahim1) [reply app/src/main/res/layout/activity_main.xml:9]: > increase font size

..

**Comment by MostafaMohamed2002** [comment]
fix text view size

**Comment by MostafaMohamed2002** [comment]
test

**Pull request merged**